wps_shared/wps_shared/db/crud/auto_spatial_advisory.py

```diff
@@ -176,6 +178,24 @@ async def get_sfms_mixed_fuel_type(session: AsyncSession) -> SFMSFuelType:
 
     return result.scalar_one()
 
+async def get_min_wind_speed_hfi_thresholds(session: AsyncSession, run_type: RunTypeEnum, run_datetime: datetime, for_date: date) -> dict[int, AdvisoryHFIWindSpeed]:
+    """
+    Retrieve min wind speeds for each hfi thresholds
+    """
+    stmt = select(AdvisoryHFIWindSpeed)\
+        .join(RunParameters, AdvisoryHFIWindSpeed.run_parameters == RunParameters.id)\
+        .join(Shape, AdvisoryHFIWindSpeed.advisory_shape_id == Shape.id)\
```

Comment:
It doesn't look like anything is being done with `Shape` so could this join be removed?